.github/workflows/bump_version.yml

```diff
@@ -0,0 +1,39 @@
+name: Bump Version
+
+jobs:
```

Comment:
I think we're missing this 
https://github.com/nimblehq/elixir-templates/blob/0f30d704740276815f54c4b4488d4dece7934141/.github/workflows/bump_version.yml#L3-L9